### PR TITLE
Public controllers for PayButton and Apps / POS

### PR DESCRIPTION
**Pull request by rockstardev**
Handling #258

### CODE REVIEW

**Comment by NicolasDorier** [comment]
@rockstardev I also wanted a small secret ID in the button. Because the button is allowing anyone knowing the storeId to create infinite amount of invoice.

This can be like "ApiKey" which can be rotated, which would be inside the Forms.

**Comment by NicolasDorier** [comment]
OR, it can be done by adding a screen to ask the user to activate Pay Button before using it.

**Comment by rockstardev** [comment]
Hey @NicolasDorier, I've added Enabling/Disabling of PayButton... review commits

File: BTCPayServer/Controllers/PublicController.cs [hunk 1]
```diff
@@ -30,9 +30,15 @@ public async Task<IActionResult> PayButtonHandle(string storeId, [FromForm]PayBu
             var store = await _StoreRepository.FindStore(storeId);
             if (store == null)
                 ModelState.AddModelError("Store", "Invalid store");
+            else
```
Comment (NicolasDorier) [inline BTCPayServer/Controllers/PublicController.cs]: `else` uneeded

**Comment by NicolasDorier** [comment]
testing it right now

File: BTCPayServer/Controllers/PublicController.cs (hunk 1 above)
Comment (rockstardev) [reply BTCPayServer/Controllers/PublicController.cs]: If `store` is null you'll trigger `store.GetStoreBlob()` line and NullException will get thrown. That's why I added `else`.

**Comment by NicolasDorier** [comment]
Merged with some of my changes on top

**Pull request closed without merging**